src/server/response.cpp

```diff
@@ -227,6 +227,13 @@ void ContentResponse::inject_externallinks_blocker()
     script_tag);
 }
 
+void ContentResponse::inject_root_link(){
+  m_content = prependToFirstOccurence(
```

Comment:
Shouldn't this be `appendToFirstOccurence()`?

Reply:
Prepend is actually adding it just after the head tag so that any module can use it and we don't have to move it up again if some other module needs it which is above this link.

Reply:
@MananJethwani "append" is add after and "prepend" is add before. If it does not work like this, this is a bug. Here you add a `<link>` node before the `<head>` node, which is wrong. I wonder if you have tested accurately the result! Actually this seems double wrong, it should be prepend to the `</head>` tag, for the same reason as for #322.

Reply:
I confirm that it was working fine, it was adding link to start of head(just after head tag before all other content of head tag) I was as well a bit confused but thought that's how it is supposed to work as a similar function is present in [jQuery(named prepend)] (https://api.jquery.com/prepend/#:~:text=prepend(%20function%20)&text=A%20function%20that%20returns%20an,of%20the%20element%20as%20arguments.)

That works exactly the same as this one is working.

Reply:
`prependToFirstOccurence()` works differently from the jQuery `prepend()`. `appendToFirstOccurence()`/`prependToFirstOccurence()` know nothing about HTML and don't operate on HTML nodes/elements. They (should) insert the specified string relative to the first occurrence of the specified regex.